### PR TITLE
Update to v1.7.9, refresh with grayskull

### PR DESCRIPTION
**Pull request by rxm7706**
Added myself as a maintainer, please feel free to remove me, if you don't need assistance.  Converted to noarch - colorama is listed as needed for windows only, but available for osx & linux

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@dbast @jkornblum 
Could you please review and merge. 
PR description has all changes
Thank you and Regards.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
 requirements:
   host:
+    - python >=3.8
     - pbr >=2.0.0
     - pip
-    - python
     - setuptools
   run:
-    - colorama >=0.3.9  # [win]
-    - GitPython >=1.0.1
```
Comment (dbast) [inline recipe/meta.yaml:28]: the baseline extra needs GitPython>=3.1.30, see https://github.com/PyCQA/bandit/blob/main/setup.cfg#L39

Please keep python first here and the rest in alphabetical order

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Review by dbast — APPROVED** [review]

**Comment by dbast** [comment]
@rxm7706 thanks!

**Pull request merged**